### ISSUE TITLE
Mage dots dont show damage numbers

### ISSUE THREAD

**Issue by imminence** [Justw8/NameplateSCT#63]
Mage dots dont show damage numbers. While eg Evoker works correctly. While eg Evoker works correctly

![image](https://github.com/user-attachments/assets/8a24e716-ce19-4a52-b3f2-209cb285080f)

**Comment by Justw8** [comment]
Do you have a minimal damage set?

**Comment by imminence** [comment]
No. Checked with evoker, it also has a problem. Dot from Hero talent also dont show damage, but from Fire Breath it does

**Comment by Justw8** [comment]
I have an evoker myself at max level to check this more, what specific hero ability (by name) @imminence

**Comment by imminence** [comment]
@Justw8 Enkindle, but i can see that it has started to work. 
But what about mages? it is still bugged. Frostfire bolt may not show at all or pop 1-2 dot numbers out of 10. The same goes for meteor burn dot.

Additionally this candle trinket works incorrect, only 1 source sows correct UI and other 2 show bliz UI eventhough its turned off
![image](https://github.com/user-attachments/assets/d3c655cd-0949-4329-8a5a-b5d780742134)

**Comment by Justw8** [comment]
The numbers in your screenshot are all NSCT. just missing an icon (no icon for the spell perhaps).
